experimental/packages/opentelemetry-configuration/package.json

```diff
@@ -40,16 +39,17 @@
     "access": "public"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.9.0",
     "@opentelemetry/core": "2.1.0",
     "yaml": "^1.10.2"
   },
+  "peerDependencies": {
+    "@opentelemetry/api": "^1.9.0"
+  },
```

Comment:
note: this was failing on the `peer-api-check` - I'm not sure why we did not include it for experimental packages, it should also apply to these.